CMakeLists.txt

```diff
@@ -32,11 +34,25 @@ endif()
 # casadi seems to compile without the newer versions of std::string
 add_compile_definitions(_GLIBCXX_USE_CXX11_ABI=0)
 
+# Add pybind11
 if(NOT PYBIND11_DIR)
   set(PYBIND11_DIR pybind11)
 endif()
-add_subdirectory(${PYBIND11_DIR})
+find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+find_package(pybind11 CONFIG REQUIRED)
+
+# if(DEFINED ENV{CIBUILDWHEEL})
+#   set(BUILD_IDAKLU ON)
+# elseif(DEFINED ENV{BUILD_IDAKLU})
+#   set(BUILD_IDAKLU "$ENV{BUILD_IDAKLU}")
+# else()
+#   set(BUILD_IDAKLU OFF)
+# endif()
 
+# message(STATUS "BUILD_IDAKLU is set to: ${BUILD_IDAKLU}")
+
+# if(BUILD_IDAKLU)
+message(STATUS "Building with IDAKLU support")
```

Comment:
```suggestion
```

I think this is obvious, since it makes sense that CMake will be invoked only if something has to be compiled, which in this case is the IDAKLU solver.

Reply:
I thought it would work well with the UX aspect. I will remove it 👍